tests/test_views.py

```diff
@@ -168,12 +168,14 @@ def test_bookmarklet_view(bukudb, client, exists, uri, tab, args):
     (True, '', 'Some description'),
     (False, 'Some title', ''),
     (False, '', 'Some description'),
+    (None, 'Some title', ''),
+    (None, '', 'Some description'),
```

Comment:
Now fetching behaviour is tested for truthy parameter, falsey parameter, and unsupplied value.